dotcom-rendering/src/web/components/Callout/MessageUs.tsx

```diff
@@ -1,83 +1,133 @@
 import { LinkButton } from '@guardian/source-react-components';
+import type { FC } from 'react';
+import type { CalloutContactType } from '../../../types/content';
 
-const calloutPrimaryButton = css`
+const calloutPrimaryButtonStyles = css`
 	width: 100%;
 	justify-content: center;
-	margin-bottom: ${space[4]}px;
+	margin-top: ${space[4]}px;
+
+	:last-of-type {
+		margin-bottom: ${space[6]}px;
+	}
 `;
 
-const descriptionStyles = css`
+const infoStyles = css`
 	a {
 		color: ${brand[500]};
-		border-bottom: 1px solid ${brand[500]};
-		text-decoration: none;
-	}
-	padding-bottom: ${space[4]}px;
-	${textSans.small()}
-
-	p {
-		margin-bottom: ${space[3]}px;
 	}
-	margin-top: ${space[2]}px;
+	margin-bottom: ${space[2]}px;
+	${textSans.xsmall()};
 `;
 
-const WHATSAPP_GUIDANCE_URL =
-	'https://www.theguardian.com/info/2015/aug/12/whatsapp-sharing-stories-with-the-guardian';
-const SECURE_DROP_URL = 'https://www.theguardian.com/securedrop';
-const WHATSAPP_SIGNAL_NUMBER = '+447766780300';
+const capitalizeFirstLetter = (s: string): string =>
+	s.charAt(0).toUpperCase() + s.slice(1);
 
-const OPEN_WHATSAPP_URL = `https://wa.me/${WHATSAPP_SIGNAL_NUMBER}`;
-const OPEN_SIGNAL_URL = `https://signal.me/#p/${WHATSAPP_SIGNAL_NUMBER}`;
+export const formatContactNumbers = (
+	contacts: CalloutContactType[],
+): string => {
+	const contactNumbers = new Map<string, string[]>();
 
-export const MessageUs = () => {
-	return (
-		<div className="js-message-us-tab">
-			<p css={descriptionStyles}>
-				You can contact us on WhatsApp or Signal at{' '}
-				{WHATSAPP_SIGNAL_NUMBER}. For more information, please see our
-				guidance on{' '}
-				<a
-					data-ignore="global-link-styling"
-					href={WHATSAPP_GUIDANCE_URL}
-				>
-					contacting us via WhatsApp
-				</a>
-				. For true anonymity please use our{' '}
-				<a data-ignore="global-link-styling" href={SECURE_DROP_URL}>
-					SecureDrop
-				</a>{' '}
-				service instead.
-			</p>
-			<LinkButton
-				data-ignore="global-link-styling"
-				cssOverrides={calloutPrimaryButton}
-				type="submit"
-				priority="primary"
-				href={OPEN_WHATSAPP_URL}
-				rel="noopener"
-				target="blank"
-				nudgeIcon={false}
-			>
-				Message us on WhatsApp
-			</LinkButton>
-			<LinkButton
+	// Group each contact by its value, so we can display multiple names for the same number.
+	contacts.forEach(({ name, value }) => {
+		if (!contactNumbers.has(value)) contactNumbers.set(value, []);
+		contactNumbers.get(value)?.push(name);
+	});
+
+	// Join the names for each number together,
+	// and then join all the numbers together into a readable string.
+	return [...contactNumbers.entries()].reduce(
+		(acc, [value, names], index) => {
+			const namesString = names
+				.map((name) => capitalizeFirstLetter(name))
+				.join(' or ');
+			return (
+				acc + (index === 0 ? '' : ' or ') + `${namesString} at ${value}`
+			);
+		},
+		'Contact us on ',
+	);
+};
```

Comment:
This logic looks great and would benefit from some unit tests if you have time. But not at all a blocker :)